.github/workflows/build-and-test.yml

```diff
@@ -119,7 +119,7 @@ jobs:
         run: |
           # uninstall InVEST if it was already in the restored cache
           python -m pip uninstall -y natcap.invest
-          python -m build --wheel
+          NATCAP_INVEST_GDAL_LIB_PATH="$CONDA_PREFIX/Library" python -m build --wheel

```

Comment:
I see we're repeating this new environment variable in a couple places, so would it interfere with anything to move it to the `env` block so we only need to define it once?  I'm pretty sure we can to a top-level `env:` mapping to allow the env var to propagate across all of the different sections.

Reply:
Nope, I see now that we won't know what `$CONDA_PREFIX` is going to be until after we're in the activated environment, so this seems like the most direct way to define it.  👍